### ISSUE TITLE
[Dropdown] Misspelled ARIA label

### ISSUE THREAD

**Issue by ardokirsipuu** [TEDI-Design-System/angular#255]
It should be `aria-haspopup` instead of `aria-has-popup` at https://github.com/TEDI-Design-System/angular/blob/d308a541ca46fabba270ef267c497fd3d44d4472/community/components/overlay/dropdown/dropdown-trigger/dropdown-trigger.directive.ts#L8

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-haspopup
